server/handlers/assessment.js

```diff
@@ -0,0 +1,64 @@
+const AWS = require('aws-sdk');
+var ep = new AWS.Endpoint(process.env.DYNAMO_ENDPOINT);
+const dynamoDB = new AWS.DynamoDB({region: 'eu-west-2', endpoint: ep});
```

Comment:
Just putting a question out here, would it be beneficial to memoize this ddb instance? 

I know performance for this app isn't going to be affected but just throwing it out there :)

Reply:
actually, forget about it. Just a note for future if we do performance related work.